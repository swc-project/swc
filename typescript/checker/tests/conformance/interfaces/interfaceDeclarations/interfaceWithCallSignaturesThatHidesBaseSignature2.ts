interface Foo {
    (): { a: number; b: number };
}

interface Derived extends Foo { // error
    (): { a: number };
}

var d: Derived;
var r = d();