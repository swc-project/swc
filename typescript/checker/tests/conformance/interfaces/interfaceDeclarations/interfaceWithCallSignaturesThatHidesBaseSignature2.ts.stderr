error: ShouldIncludeUndefinedType {
    span: Span {
        lo: BytePos(
            132,
        ),
        hi: BytePos(
            133,
        ),
        ctxt: #0,
    },
}
 --> $DIR/tests/conformance/interfaces/interfaceDeclarations/interfaceWithCallSignaturesThatHidesBaseSignature2.ts:9:5
  |
9 | var d: Derived;
  |     ^

